app/assets/stylesheets/components/_card.scss

```diff
@@ -4,17 +4,20 @@
   box-shadow: 0 0 10px rgba(0,0,0,0.2);
   margin: 10px 0;
   position: relative;
-  a {
+  a, .chart-title {
     color: #484848;
     text-decoration: none;
     font-size: 22px;
     font-weight: bolder;
     transition: all 0.25s ease;
   }
-  &:hover a {
+  &:hover .chart-title {
```

Comment:
I'm not sure if it's good practice to use &:hover when on line 7 your applying css to two different elements. It's kinda confusing. Perhaps there's no need to declare the a tag here either as we will have to do that in a general place for the entire website.